modules/ROOT/pages/tips-and-tricks.adoc

```diff
@@ -20,7 +20,31 @@ This will behave as follows
 * When run from the host, returns an exit code of 1
 * When run from a Toolbx container, returns an exit code of 0 and prints the
   current Toolbx containers name to the console
-
+  
+If a more automated solution is your preference the following added to your `~/.bashrc` will change your bash prompt to
+include "[toolbox <name>]":
+
+```
+function is_toolbox() {
+    # $HOSTNAME is assigned to "toolbox" when running in a toolbox container.
+    if [ "${HOSTNAME}" == "toolbox" ]
```

Comment:
I would prefer we use the presence of `/run/.toolboxenv` as above as a marker of being in a toolbox as this is potentially the stable interface. There is a discussion about changing the way hostname are set for toolbox and would break this.

Reply:
Updating my local branch now.